app/views/events/edit.html.haml

```diff
@@ -9,3 +9,11 @@
   .row
     .span16
       = render 'form'
+  - if @other_conferences.any?
+    = semantic_form_for(@event, :url => copy_to_conference_event_path) do |f|
```

Comment:
This doesn't work anymore since the rebase. Don't merge this yet, will push a new version.